### PR TITLE
Fix WalletSimple sequenceID handling

### PR DESCRIPTION
**Pull request by marktoda**
The WalletSimple multisig contract validates a sequence ID with each
signature to protect against signature replays. In the previous version
of the contract, this sequenceId was validated against a moving window
of the last 10 ids. This allowed for some flexibility with transaction
ordering, while avoiding the same id being used twice, and avoiding
keeping much state around. In the new version, we deemed this
unnecessary and simplified it to a purely increasing value. However,
it turns out that the ordering flexilibity is actually desirable and
worth the cost. This commit brings back the original moving window
implementation.

### CODE REVIEW

File: contracts/Batcher.sol [hunk 1]
```diff
@@ -17,94 +18,114 @@ pragma solidity 0.7.5;
  */
 
 contract Batcher {
-    event BatchTransfer(address sender, address recipient, uint256 value);
-    event OwnerChange(address prevOwner, address newOwner);
-    event TransferGasLimitChange(uint256 prevTransferGasLimit, uint256 newTransferGasLimit);
+  event BatchTransfer(address sender, address recipient, uint256 value);
```
Comment (marktoda) [inline contracts/Batcher.sol:21]: Just spacing changes.. ran linter

File: contracts/Forwarder.sol [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity 0.7.5;
-import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
-import "./ERC20Interface.sol";
+import '@uniswap/lib/contracts/libraries/TransferHelper.sol';
```
Comment (marktoda) [inline contracts/Forwarder.sol:3]: Just lint changes to this file

**Comment by marktoda** [comment]
Note reviewing commit by commit will be easier - first commit is just linter fix, second commit is actual change

**Comment by marktoda** [comment]
For comparison to the previous implementation: https://github.com/BitGo/eth-multisig-v2/blob/8544002d078d6bebcff4017fda7b40a534087bbe/contracts/WalletSimple.sol#L274

**Review by barathcj — APPROVED** [review]
lgtm

**Pull request merged**